lib/helpers/dockerCompose.js

```diff
@@ -2,10 +2,10 @@
 
 require('shelljs/global');
 
-module.exports = function() {
-  if(which('docker-compose')) {
+module.exports = function () {
+  if (which('docker-compose')) {
```

Comment:
I like this, `if` is a cyclomatic.  I think the rule should be cyclomatic keywords are surrounded by a space.